samza-kafka/src/main/scala/org/apache/samza/config/KafkaConfig.scala

```diff
@@ -292,7 +292,8 @@ class KafkaConfig(config: Config) extends ScalaMapConfig(config) {
   // The method returns a map of storenames to changelog topic names, which are configured to use kafka as the changelog stream
   def getKafkaChangelogEnabledStores() = {
```

Comment:
Could you add a return value to this method to clarify that it is returning a `util.HashMap` (as opposed to a scala `Map`)?

Reply:
he he. Intelij shows it :) in grayed out script. I will add. Makes sense.